docs/01_technical-overview/index.md

```diff
@@ -5,20 +5,20 @@ In general, there are two objects that are used to interact with a blockchain vi
 ## JsonRpc
 The `JsonRpc` object is typically used when signing is not necessary.  Some examples include [getting block information](how-to-guides/00_how-to-get-block-information.md), [getting transaction information](how-to-guides/02_how-to-get-transaction-information.md), or [getting table information](how-to-guides/09_how-to-get-table-information.md).  
 
-The requests made by the `JsonRpc` object will either use a built-in `fetch` library, or [the `fetch` library passed in by the user](basic-usage/01_commonjs.md) to issue requests to the endpoint specified when instantiating the `JsonRpc` object.  When the various methods ([get_abi](https://github.com/EOSIO/eosjs/blob/master/src/eosjs-jsonrpc.ts#L66), [get_account](https://github.com/EOSIO/eosjs/blob/master/src/eosjs-jsonrpc.ts#L71), [get_block_header_state](https://github.com/EOSIO/eosjs/blob/master/src/eosjs-jsonrpc.ts#L76), etc) of the `JsonRpc` object are invoked, the calls are delegated to the `JsonRpc` object's [fetch function](https://github.com/EOSIO/eosjs/blob/master/src/eosjs-jsonrpc.ts#L42-L63), which in turn, delegate the requests to the `fetch` library.
+The requests made by the `JsonRpc` object will either use a built-in `fetch` library, or [the `fetch` library passed in by the user](basic-usage/01_commonjs.md) to issue requests to the endpoint specified when instantiating the `JsonRpc` object.  When the various methods ([get_abi](https://github.com/EOSIO/eosjs/blob/4b3413d6170c07682be06e8400c668688d115c0c/src/eosjs-jsonrpc.ts#L119), [get_account](https://github.com/EOSIO/eosjs/blob/4b3413d6170c07682be06e8400c668688d115c0c/src/eosjs-jsonrpc.ts#L123), [get_block_header_state](https://github.com/EOSIO/eosjs/blob/4b3413d6170c07682be06e8400c668688d115c0c/src/eosjs-jsonrpc.ts#L145), etc) of the `JsonRpc` object are invoked, the calls are delegated to the `JsonRpc` object's [fetch function](https://github.com/EOSIO/eosjs/blob/4b3413d6170c07682be06e8400c668688d115c0c/src/eosjs-jsonrpc.ts#L77), which in turn, delegate the requests to the `fetch` library.
 
 ## Api
 The `Api` object is typically used when transacting on an EOSIO-based blockchain.  Some examples include [staking](how-to-guides/03_how-to-stake.md), [creating an account](how-to-guides/05_how-to-create-an-account.md), or [proposing multi-sig transactions](how-to-guides/13_how-to-propose-a-multisig-transaction.md).
 
-The typical use of the `Api` object is to call its [`transact` method](https://github.com/EOSIO/eosjs/blob/master/src/eosjs-api.ts#L214-L254).  This method performs a number of steps depending on the input passed to it:
+The typical use of the `Api` object is to call its [`transact` method](https://github.com/EOSIO/eosjs/blob/release/22.1.x/src/eosjs-api.ts#L334).  This method performs a number of steps depending on the input passed to it:
```

Comment:
Probably use a commit hash on this link as well since release/22.1.x branch is still changing and if any changes are made before L334, it won't link to the correct line anymore.